### PR TITLE
Supermatter Delamination Balance Changes (Real)

### PR DESCRIPTION
**Pull request by Lufferly**
## About The Pull Request

lord forgive me I fucked up the merge conflict

The supermatter delamination countdown timer (how long it takes to go boom-boom after hitting 0 integrity) has been lowered from 30 seconds to 13 seconds.
Removing a sliver from the supermatter, the traitor objective, will further lower that down to 3 seconds.
Changes the wording on the mood effects from the supermatter delaminating slightly.
The crystal uses SPAN_COMMAND on its final countdown, which means it talk bigger.

## Why It's Good For The Game

Currently I feel that the supermatter delamination countdown overstays its welcome. Ideally it provides tension to get the hell out, or perhaps to make a risky last second play to try and save the supermatter. However right now its at 30 seconds, which gives no danger of staying in engineering right up to integrity 0, and keeps the tension at a high note for too long, almost to the point of awkwardness. 13 seconds is a good balance between get-the-fuck-out while still giving some leeway for engineers to escape and crewmembers to jump in lockers, and feels quick enough to give that danger that the supermatter should provide.
Additionally, removing a sliver from the supermatter lowers the cooldown to 3 seconds. Right now this is one of the harder tasks a traitor can be tasked with, while giving relatively little payoff sabatoge-wise. To the point where I have seen engineers just let the traitor do it, as the debuff it gives to the supermatter is minor. Now it makes the supermatter delaminate almost immediately after hitting 0 integrity, which means it will likely catch some engineers in the blast if a traitor did it stealthy. This also makes it more risky to try and fix a delamination if the engineering/security team did not stop the sliver from being removed. All meaning succeeding at this task should be more rewarding and damaging.
Finally the mood descriptions for the mood effects you get when a supermatter delaminates have been changed. Currently they are pretty gamey, and represent what the player might be thinking more than their character. Additionally they were not very descriptive of where they came from, which could be confusing.

## Changelog

:cl: Seven
balance: The supermatter delamination countdown has been lowered from 30 to 13 seconds
balance: Removing a sliver from the supermatter further lowers that down to 3 seconds
balance: The supermatter crystal uses bigger text on its final countdown
spellcheck: Some supermatter delamination related mood descriptions have been edited to explain the mood effect better
/:cl:

### CODE REVIEW

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -43,12 +43,12 @@
...
 
 /datum/mood_event/delam //SM delamination
-	description = "Those goddamn engineers can't do anything right..."
+	description = "Ever since that supermatter delamination, my head has been killing me..."
 	mood_change = -2
 	timeout = 4 MINUTES
 
 /datum/mood_event/cascade // Big boi delamination
-	description = "The engineers have finally done it, we are all going to die..."
+	description = "A supermatter cascade... this is the end..."
```
Comment (carlarctg) [inline code/datums/mood_events/generic_negative_events.dm:51]:
```suggestion
	description = "I never thought I'd see a resonance cascade, let alone be near one..."
```

funny reference

**Comment by SimplySmiley123** [comment]
Again, like in previous try at this, 13 seconds still feels too little.

**Comment by YakumoChen** [comment]
👎 I liked the old mood texts, they were funny and still true. Don't know why the character in game wouldn't be thinking the same thing.

**Comment by ZeroKelvn** [comment]
Now that there's an achievement for saving the Supermarket after it's gone critical, this pretty much guts it. Never seen anyone go for it before, you're just incentivizing Engineering to give up on the engine and force a shuttle call.

File: code/modules/power/supermatter/supermatter.dm [hunk 2]
```diff
@@ -501,10 +504,23 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
...
 
-	for(var/i in SUPERMATTER_COUNTDOWN_TIME to 0 step -10)
+	var/delamination_countdown_time = SUPERMATTER_COUNTDOWN_TIME
+	// If a sliver was removed from the supermatter, the countdown time is significantly decreased
+	if (supermatter_sliver_removed == TRUE)
+		delamination_countdown_time = SUPERMATTER_SLIVER_REMOVED_COUNTDOWN_TIME
+		radio.talk_into(
+			src,
+			"WARNING: Projected time until full crystal delamination significantly lower than expected. \
+			Please inspect crystal for structural abnormalities or sabatoge!",
```
Comment (Shadow-Quill) [inline code/modules/power/supermatter/supermatter.dm:518]:
```suggestion
			Please inspect crystal for structural abnormalities or sabotage!",
```

**Review by ZephyrTFA — APPROVED** [review]

**Comment by Googles-Hands** [comment]
Well, it's been approved but hey, I'll still put my opinion out there. The mood texts should remain as is, it's an IC flavour on how and why your character feels bad, it doesn't need to explain exactly whats happening.

As for the countdown, 13 seconds is fine and the sliver stealing is hard enough on normal pop to grant the 3 seconds.

**Comment by RikuTheKiller** [comment]
It's been merged, but the texts should've been left alone.
Also can it just be 15 seconds instead of 13? AAAAA

**Pull request merged**